files/es/glossary/base64/index.md

```diff
@@ -17,68 +17,59 @@ La función `atob()` decodifica una cadena de datos que ha sido codificada usand
 
 Ambas funciones trabajan sobre cadenas de texto. Si desea trabajar con [ArrayBuffers](/es/docs/Web/JavaScript/Referencia/Objetos_globales/ArrayBuffer), lea [este párrafo](/es/docs/Web/API/WindowBase64/Base64_codificando_y_decodificando$edit#Solution_.232_.E2.80.93_rewriting_atob%28%29_and_btoa%28%29_using_TypedArrays_and_UTF-8).
 
-<table>
-  <tr>
-    <td>
-
-- [`data` URIs](https://developer.mozilla.org/en-US/docs/data_URIs)
+- [`data` URIs](/en-US/docs/data_URIs)
   - `Los URIs de` datos, definidos por [RFC 2397](https://tools.ietf.org/html/rfc2397), permiten a los creadores de contenido introducir pequeños ficheros en línea en documentos.
-  
+
 - [Base64](https://es.wikipedia.org/wiki/Base64)
 
   - Artículo en Wikipedia sobre el sistema de codificación Base64.
 
-- [`atob()`](https://developer.mozilla.org/es/docs/Web/API/atob "atob()")
+- [`atob()`](https://developer.mozilla.org/es/docs/Web/API/atob)
```

Comment:
We're clobbering the link titles here in this page. Is that acceptable or a known limitation of the linter?

Reply:
This was done on the source English files after the conversion because these titles were mostly duplicates of the link text

Reply:
It's not something that Markdownlint is doing, but a cleanup I rolled into these PRs (from a previously open PR)

Reply:
I understand, thanks